esp-box/esp-box.c

```diff
@@ -368,5 +371,16 @@ esp_err_t bsp_button_init(const bsp_button_t btn)
 
 bool bsp_button_get(const bsp_button_t btn)
 {
-    return !(bool)gpio_get_level(btn);
+#if (CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0)
+    if (btn == BSP_BUTTON_MAIN) {
+        uint8_t home_btn_val = 0x00;
+        assert(tp);
+
+        esp_lcd_touch_get_button_state(tp, 0, &home_btn_val);
+
+        return home_btn_val ? true : false;
+    } else {
+        return !(bool)gpio_get_level(btn);
```

Comment:
I think, this shopuldn't be inside the `#if (CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0)`. 
My suggestion:
```
    if (btn == BSP_BUTTON_MAIN) {
#if (CONFIG_ESP_LCD_TOUCH_MAX_BUTTONS > 0)
        uint8_t home_btn_val = 0x00;
        assert(tp);

        esp_lcd_touch_get_button_state(tp, 0, &home_btn_val);
        return home_btn_val ? true : false;
#else
       ESP_LOGE(TAG, "Some error message.....");
       return false;
#endif
    } else {
        return !(bool)gpio_get_level(btn);
    }
```

Reply:
Good suggestion, please check the latest one again, thanks.